lib/debug/color.rb

```diff
@@ -64,6 +64,16 @@ def color_pp obj, width
       end
     end
 
+    # colorize documentation of h[elp] and ? instructions
+    def colorize_help(desc)
```

Comment:
The `Color` module is for generic coloring helpers. So I think this method shouldn't be added here.

Reply:
I removed the method from color.rb, and now it is using a proc inside show_help method in session.rb.

Thank you.